guide/english/css/css3-shadow-effects/index.md

```diff
@@ -67,6 +67,13 @@ The `box-shadow` property can take up to six values:
 ```
 ![Box-shadow examples](https://raw.githubusercontent.com/nawnaw7/FCC-guides-CSS3-shadows-images/master/CSS3%20Shadow%20Effects%20Images/box-shadows.png)
 
+##### Box Shadow As A Border
+You can also use the `box-shadow` property to create a border around an element. This allows you to have more control over the border style of an element, rather than using the `border` property:
+```css
+.div {
```

Comment:
If you're using a class selector, please use something other than `div`. As `div` is an HTML element, it's not ideal to use it as a classname.